src/install/fetch.ts

```diff
@@ -167,13 +167,25 @@ type FetchToStoreOptions = {
  */
 function fetchToStoreCached (opts: FetchToStoreOptions): Promise<void> {
   return memoize(opts.fetchLocks, opts.resolution.id, async function () {
-    if (opts.force || !await exists(opts.target)) {
-      await rimraf(opts.target)
+    const target = opts.target
+    const targetStage = `${opts.target}-stage`
```

Comment:
Hm... right. I'll change `-target` suffix to `_target` which doesn't pass [semver validator](http://jubianchi.github.io/semver-check/).
 
Also I don't understand the failure on node v4, `rimraf` is used to remove the dir but it still complains dir isn't empty.